gprofiler/profilers/java.py

```diff
@@ -466,10 +466,15 @@ def _get_extra_ap_args(self) -> str:
     def _get_ap_output_args(self) -> str:
         return f",file={self._output_path_process},{self.OUTPUT_FORMAT},{self.FORMAT_PARAMS}"
 
+    def _get_interval_arg(self, interval: int) -> str:
+        if self._mode != "alloc":
+            return f",interval={interval}"
+        return ""
```

Comment:
Choose a sane default for allocation, or pass it from frequency, but a sane default is needed

Reply:
done